CONTRIBUTING.md

```diff
@@ -99,45 +111,42 @@ Then you can run tests:
     # Run all tests
     $ make test_all
 
-    # Run unit tests
+    # Run unit tests (tests/Unit folder)
     $ make test_unit
 
-    # Run integration tests
+    # Run integration tests (tests/Integration folder)
     $ make test_integration
 
-    # Run auto-instrumentation tests
+    # Run auto-instrumentation tests (tests/AutoInstrumentation folder)
     $ make test_auto_instrumentation
 
-    # Run composer integration tests
+    # Run composer integration tests (tests/Composer folder)
     $ make test_composer
 
-    # Run distributed tracing tests
+    # Run distributed tracing tests (tests/DistributedTracing folder)
     $ make test_distributed_tracing
 
-    # Run library integrations tests
+    # Run library integrations tests (tests/Integrations folder)
     $ make test_integrations
 
     # Run web frameworks integrations tests
     $ make test_web
 
+    # Run C Tests (the ones in tests/ext)
+    $ make test_c
+
 In order to run the `phpt` tests for the php extension:
 
 ```bash
 $ composer test-ext
 ```
 
-### PHP linting
```

Comment:
This section is still relevant, we just removed the C code style, not the PHP code style.